jhipster-framework/src/main/java/io/github/jhipster/config/cache/CacheEntryListenerLogger.java

```diff
@@ -0,0 +1,51 @@
+package io.github.jhipster.config.cache;
+
+import javax.cache.event.CacheEntryCreatedListener;
+import javax.cache.event.CacheEntryEvent;
+import javax.cache.event.CacheEntryExpiredListener;
+import javax.cache.event.CacheEntryListenerException;
+import javax.cache.event.CacheEntryRemovedListener;
+import javax.cache.event.CacheEntryUpdatedListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A custom cache entry listener.
+ */
+public class CacheEntryListenerLogger implements CacheEntryCreatedListener<Object, Object>, CacheEntryUpdatedListener<Object, Object>, CacheEntryExpiredListener<Object, Object>, CacheEntryRemovedListener<Object, Object> {
+
+    private static final Logger log = LoggerFactory.getLogger(CacheEntryListenerLogger.class);
+    
+    /** {@inheritDoc} */
+    @Override
+    public void onCreated(Iterable<CacheEntryEvent<? extends Object, ? extends Object>> events) throws CacheEntryListenerException {
+        for (CacheEntryEvent<? extends Object, ? extends Object> entryEvent : events) {
```

Comment:
You should shield the for loops with a `isInfoEnabled()`. And I do think it should be at debug level. Or Configurable.